addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -505,17 +505,18 @@ options.registry.layout_column = options.Class.extend({
             });
         }
 
-        this._resizeColumns();
+        this._resizeColumns(count);
         this.trigger_up('cover_update');
     },
     /**
      * Resizes the columns so that they are kept on one row.
      *
      * @private
      */
-    _resizeColumns: function () {
+    _resizeColumns: function (count) {
+        count = count || 0;
         var $columns = this.$target.children();
-        var colsLength = $columns.length;
+        var colsLength = $columns.length + count;
```

Comment:
The parameter has to be documented. Actually it would make more sense to me that this independent function here receives `$columns.length + count`, default to `$columns.length`. Also in the call above, it also make more sense to compute that value before the `trigger_up`, to not rely on them being synchronous or asynchronous (that last point is not that important though).

Reply:
Done